.circleci/config.yml

```diff
@@ -1196,24 +1214,24 @@ jobs:
       compile_only:
         type: integer
         default: 0
-      nodejs_version:
+      image:
         type: string
-        default: current
+        default: cimg/node:current
       resource_class:
         type: string
         default: small
       python2:
         type: boolean
         default: false
     docker:
-      - image: cimg/node:<<parameters.nodejs_version>>
-    resource_class: <<parameters.resource_class>>
+      - image: << parameters.image >>
+    resource_class: << parameters.resource_class >>
     # NOTE: Each external test runs up to 6 independent settings presets. If parallelism is higher than
     # actual preset count, some runs will exit immediately. If it's lower, some runs will get more than one preset.
     parallelism: 6
     environment:
       TERM: xterm
-      COMPILE_ONLY: <<parameters.compile_only>>
+      COMPILE_ONLY: << parameters.compile_only >>
```

Comment:
We should settle on a single style for this. I was generally skipping the spaces because it looks more compact this way and it's not bash so it does not need spacing around parentheses.

Reply:
I just used the pattern that was used in other places, but I'm fine with either way.